rueidiscompat/pipeline_test.go

```diff
@@ -601,6 +601,7 @@ func TestPipeliner(t *testing.T) {
 		p.JSONToggle(ctx, "1", "1")
 		p.JSONType(ctx, "1", "1")
 		p.SlaveOf(ctx, "NO", "ONE")
+		p.SlowLogGet(ctx, 1)
```

Comment:
You also need to update L612 and L620:
<img width="635" alt="image" src="https://github.com/user-attachments/assets/af0dc011-c195-490c-a851-c5796319bc66" />